src/main/java/com/tileman/TilemanModeConfig.java

```diff
@@ -83,6 +83,9 @@ default boolean automarkTiles() {
         return false;
     }
 
+    @Range(
```

Comment:
why have a negative warning limit?

Reply:
I thought if people are able to have negative tiles maybe they want a negative warning limit (not likely but no reason to restrict people imo)

Reply:
I think this is just confusing though for 99% of use cases. Like people will wonder what a negative limit means. Plus if people are constantly in a deficit, it seems like the warning should just be red since they don't have any tiles to place